### ISSUE TITLE
Logging Object Depth

### ISSUE THREAD

**Issue by rjhilgefort** [raine/treis#3]
I often need a bit more depth when logging big objects. Could we increase the depth? If not, can you  recommend a workaround instead?

**Comment by raine** [comment]
In which case exactly are you having trouble? It should be logging objects fully.

https://github.com/raine/treis/blob/master/lib/utils.js#L6

edit: I'm noticing `maxArrayLength` is not set, so maybe you're talking about large arrays not being logged fully?

**Comment by rjhilgefort** [comment]
Possibly. It seems to be a depth issue when any size array is involved. Below is the spot in my React app where I saw the issue. I realize that this doesn't do much for you to be able to reproduce, but it does show the issue. If need be, I can spend a few minutes constructing a case in a REPL.

Here's the line without any `treis` logging. I'm using lenses, so just know that the `L.set` will receive two arguments `(prevState, props)`, both of which are objects. The response is the newly updated state (another object).

```js
this.setState(
  L.set(fieldFilterValuesL(fieldName), event.target.value)
);
```

If I wrap this in `treis` like this:

```js
this.setState(
  tr(L.set(fieldFilterValuesL(fieldName), event.target.value))
);
```

I get the following logs in the browser console:

```
λ2 x2: { technicalInfo: 
          { filters: [ [Object], [Object], [Object] ],
            approved: false,
            __typename: 'TechnicalInfo' },
         open: true }
   1: { approvalPermission: true,
        permission: true,
        technicalInfo: 
         { filters: [ [Object], [Object], [Object] ],
           approved: false,
           __typename: 'TechnicalInfo' },
        updateTechnicalInfo: [Function] }
λ2 => { technicalInfo: 
         { filters: [ [Object], [Object], [Object] ],
           approved: false,
           __typename: 'TechnicalInfo' },
        open: true }
```

You can see that the `filters` array doesn't have it's objects expanded. Is this by design? Is there any mechanism to fully recurse the structure?

**Comment by raine** [comment]
Fixed in `v2.6.0`.

Problem was that `util-inspect` module that was used on browser side had no support for `depth` option. Back when I implemented this module, it was the only option for doing `util.inspect` like functionality on browser side. Now it's using whatever shim browserify provides, and it seems to work.

I also added color support for browser with `ansicolor` module.   https://github.com/raine/treis/commit/70dcc25e857ba93c5f8806adb926433c49fb856c

![screen shot 2018-06-15 at 12 03 58](https://user-images.githubusercontent.com/11027/41460108-365034a6-7094-11e8-98b2-80da678f11ce.png)
